pkg/daemonsetlock/daemonsetlock.go

```diff
@@ -34,11 +41,11 @@ func New(client *kubernetes.Clientset, nodeID, namespace, name, annotation strin
 }
 
 // Acquire attempts to annotate the kured daemonset with lock info from instantiated DaemonSetLock using client-go
-func (dsl *DaemonSetLock) Acquire(metadata interface{}, TTL time.Duration) (acquired bool, owner string, err error) {
+func (dsl *DaemonSetLock) Acquire(metadata interface{}, TTL time.Duration) (bool, string, error) {
 	for {
-		ds, err := dsl.client.AppsV1().DaemonSets(dsl.namespace).Get(context.TODO(), dsl.name, metav1.GetOptions{})
+		ds, err := dsl.GetDaemonSet(k8sAPICallRetrySleep, k8sAPICallRetryTimeout)
```

Comment:
is GetDaemonSet meant to be exposed, or is that an internal implementation detail?

Reply:
I followed the existing pattern of using exportable methods (capital first letter) against the exportable `DaemonSetLock` type.

I actually think go will complain if you use a private method on an exportable type struct.

(Is that what you're asking?)

Reply:
You're right. It's good.